src/app_service/service/chat/service.nim

```diff
@@ -489,15 +489,14 @@ QtObject:
         let leaveGroupResponse = status_chat.leaveGroupChat(chatId)
         self.emitUpdate(leaveGroupResponse)
 
-      discard status_chat.deactivateChat(chatId)
+      discard status_chat.deactivateChat(chatId, preserveHistory = chat.chatType == chat_dto.ChatType.OneToOne)
```

Comment:
Just wondering what happens to group chats? :thinking:

Reply:
Same behaviour as before. The history gets deleted and the chat is left. You completely unsubscribe from the chat.